CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ update DATABASECHANGELOG
 * Added custom thread pool to workaround Slick [deadlocks](https://github.com/slick/slick/issues/1274). The thread pool
 size defaults to the Slick configuration value `db.numThreads`, but may be increased up to Slick's
 `db.maxConnections`, via a new property `actionThreadPoolSize`.
+* Added support for [size](https://github.com/broadinstitute/wdl/blob/master/SPEC.md#float-sizefile-string) WDL standard library function.
```

Comment:
So as far as I know we don't have a process for when to merge `develop` -> `master` on the wdl repository.  I was under the impression that `develop` would only get merged when we decide that it's a good time for a new version of the specification